src/node_modules.cc

```diff
@@ -344,6 +359,28 @@ void BindingData::GetNearestParentPackageJSONType(
   args.GetReturnValue().Set(Array::New(realm->isolate(), values, 3));
 }
 
+void BindingData::GetPackageJSONScripts(
+    const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+  std::string path = "package.json";
```

Comment:
I have not looked deeply at the issue, but could this be a std::string_view instead of a std::string?

Reply:
This is a right assessment. I've fixed the code. Thank you for the review.